src/ElvAccount.js

```diff
@@ -56,17 +58,35 @@ class ElvAccount {
    *
    * @namedParams
    * @param {number} funds - The amount in ETH to fund the new account.
-   * @cauth {string} accountName - The name of the account to set in it's wallet metadata (Optional)
-   * @cauth {string} tenantAdminsGroup - The tenant admins group ID to set for the user's wallet (Optional)
+   * @param {string} accountName - The name of the account to set in it's wallet metadata (Optional)
+   * @param {string} tenantId - The tenant ID (iten) (Optional)
    * @return {Promise<Object>} - An object containing the new account mnemonic, privateKey, address, accountName, balance
    */
-  async Create({ funds = 0.25, accountName, tenantAdminsAddress }) {
+  async Create({ funds = 0.25, accountName, tenantId }) {
     if (!this.client) {
       throw Error("ElvAccount not intialized");
     }
-    if (tenantAdminsAddress) {
-      if (!Utils.ValidAddress(tenantAdminsAddress)) {
-        throw Error(`Invalid tenant admins address: ${tenantAdminsAddress}`);
+
+    // We don't require the key is part of a tenant (for example when creating a tenant root key)
+    let tenantAdminsAddr;
+    if (tenantId != undefined) {
+      // Validate tenant ID (make sure it is not the tenant admins ID)
+      const idType = await this.client.AccessType({ id: tenantId });
+      if (idType != this.client.authClient.ACCESS_TYPES.TENANT) {
+        throw Error("Bad tenant ID");
+      }
+
+      // Find tenant admins address
+      let tenantAddr = this.client.utils.HashToAddress(tenantId);
+      try {
+        tenantAdminsAddr = await this.client.CallContractMethod({
+          contractAddress: tenantAddr,
+          methodName: "groupsMapping",
+          methodArgs: ["tenant_admin", 0],
```

Comment:
I didn't remember we had that function - though I think it's more intuitive for the client to get the group directly to avoid even more confusion about the tenantId